### PR TITLE
FitLLS feature

### PR DESCRIPTION
**Pull request by profxj**
Add masking to LLS fitting.

### CODE REVIEW

File: pyigm/surveys/utils.py [hunk 1]
```diff
@@ -58,14 +58,18 @@ def load_sys_files(inp, type, ref=None, sys_path=False, build_abs_sys=False, **k
             survey._abs_sys.append(abssys)
     else:  # tarball
         print('Loading systems from {:s}'.format(inp))
-        tar = tarfile.open(inp)
+        print('Loading systems from {:s}'.format(inp))
```
Comment (sstawins) [inline pyigm/surveys/utils.py]: Don't print twice

File: pyigm/guis/fitlls.py [hunk 2]
```diff
@@ -292,10 +292,10 @@ def setbzN(self):
             self.Nwidget.set_text(40.)
         # z
         try:
-            self.abssys_widg.all_abssys[idx].zabs = (
+            self.abssys_widg.all_abssys[idx].limits._z = (
                 float(self.zwidget.box.text()))
         except:
-            self.abssys_widg.all_abssys[idx].zabs = 2.
+            self.abssys_widg.all_abssys[idx].limits._z = 2.
```
Comment (sstawins) [inline pyigm/guis/fitlls.py:298]: Model doesn't update

**Review by sstawins — APPROVED** [review]

File: pyigm/surveys/utils.py (hunk 1 above)
Comment (profxj) [reply pyigm/surveys/utils.py]: fixed

File: pyigm/guis/fitlls.py (hunk 2 above)
Comment (profxj) [reply pyigm/guis/fitlls.py:298]: agreed

**Comment by profxj** [comment]
merging

**Pull request merged**